addons/mail/controllers/discuss/binary.py

```diff
@@ -75,6 +75,26 @@ def discuss_channel_attachment(self, channel_id, attachment_id, download=None):
             raise NotFound()
         return request.env["ir.binary"]._get_stream_from(attachment_sudo).get_response(as_attachment=download)
 
+    @http.route(
+        "/discuss/channel/<int:channel_id>/avatar_128",
+        methods=["GET"],
+        type="http",
+        auth="public",
+    )
+    def discuss_channel_avatar_128(self, channel_id):
+        channel_member_sudo = request.env["discuss.channel.member"]._get_as_sudo_from_request_or_raise(
+            request=request, channel_id=int(channel_id)
+        )
+        domain = [("id", "=", int(channel_id))]
```

Comment:
you probably don't need to cast `channel_id` as it is already typed in the route